src/pages/views/components/charts/historicalCostChart/historicalCostChart.test.tsx

```diff
@@ -57,45 +51,13 @@ test('null previous and current reports are handled', () => {
       previousInfrastructureCostData={null}
     />
   );
-  const charts = view.find(ChartArea);
+  const charts = screen.getAllByText(/cost/i);
   expect(charts.length).toBe(4);
 });
 
 test('height from props is used', () => {
-  const view = shallow(<HistoricalCostChart {...props} />);
-  expect(view.find(Chart).prop('height')).toBe(props.height);
-});
-
-test('labels formats with datum and value formatted from props', () => {
-  const view = shallow(<HistoricalCostChart {...props} />);
-  const datum: utils.ChartDatum = {
-    x: 1,
-    y: 1,
-    key: '1-1-1',
-    units: 'hrs',
-  };
-  const group = view.find(Chart);
-  group.props().containerComponent.props.labels({ datum });
-  expect(props.formatter).toBeCalledWith(datum.y, datum.units, props.formatOptions);
-  expect(view.find(Chart).prop('height')).toBe(props.height);
-});
-
-test('trend is a running total', () => {
-  const multiDayReport: OcpReport = {
-    data: [createReportDataPoint('1-15-18', 1), createReportDataPoint('1-16-18', 2)],
-  };
-  const view = shallow(<HistoricalCostChart {...props} currentCostData={multiDayReport.data} />);
-  const charts = view.find(ChartArea);
-  expect(charts.at(1).prop('data')).toMatchSnapshot('current month data');
-});
-
-test('trend is a daily value', () => {
-  const multiDayReport: OcpReport = {
-    data: [createReportDataPoint('1-15-18', 1), createReportDataPoint('1-16-18', 2)],
-  };
-  const view = shallow(<HistoricalCostChart {...props} currentCostData={multiDayReport.data} />);
-  const charts = view.find(ChartArea);
-  expect(charts.at(1).prop('data')).toMatchSnapshot('current month data');
```

Comment:
We don't always have a months worth of data. The current month have two or more data points, but not always a full month. Sometimes it's zero on the 1st of the month.

We often have a full previous month's worth of data, but that is not always the case, either.

That said, we do need at least two data points in order for the charts to draw a line. Otherwise, a single data point won't chart anything.

Reply:
Note that line charts show cumulative data by default, while the bar charts show raw daily data. You may be seeing what is done to display cumulative charts? That is, the current day is added to the previous one.

Reply:
I'll revisit

Reply:
I'd like to see at least two data points. The cumulative charts may pad data, but the daily charts won't

Reply:
it seemed like the data that was getting built was not correct, i ended up just putting a breakpoint on the props in the running app and copying over that into a file to make sure we had good data. i'm still having issue getting intl to return something meaningful for anything other than if it is a direct property of the component i'm testing... but the data in the snapshot will show the correct x/y values etc.